api/api/V1/Device.ts

```diff
@@ -1167,16 +1278,15 @@ app.get(
       });
 
       if (deviceSettings) {
-        return successResponse(response, "Device mask-regions retrieved successfully", deviceSettings.settings);
+        return successResponse(response, "Device settings retrieved successfully", deviceSettings.settings);
```

Comment:
Was this API just added to facilitate testing, or does it have another usage?

Reply:
I thought there would be instances where we may want to retrieve all the settings within deviceHistory, not just maskRegions. I have used it in my testing to prove that the mask-regions API allows the rest of the settings to persist.